src/input/Input.js

```diff
@@ -60,6 +60,8 @@ class Input extends React.Component {
   render() {
     const {
       containerStyle,
+      disabled,
+      disabledInputStyles,
```

Comment:
disabledInputStyle not disabledInputStyles.

Has to be consistent with the non-disabled prop

Reply:
Sure. I've just changed this.